meson.build

```diff
@@ -1,20 +1,25 @@
 project('sente', 'cpp',
-        default_options: ['cpp_std=c++17'], version: '0.4.2')
+        default_options: ['cpp_std=c++17'], version: '0.4.3')
 
 # ===========================================================
 # Python 3
 # ===========================================================
 
 # python 3 import for extension_module
 python_module = import('python')
-inst = python_module.find_installation(get_option('python_executable'))
+inst = python_module.find_installation()
 
 # ===========================================================
 # PyBind11
 # ===========================================================
 
 # pybind11 dependency
-pybind11_dep = dependency('pybind11')
+pybind11_dep = dependency('pybind11', required: false)
+
+if not pybind11_dep.found()
+    libpybind11 = subproject('pybind11')
+    pybind11_dep = libpybind11.get_variable('pybind11_dep')
+endif
```

Comment:
If you are using the WrapDB's pybind11.wrap then it contains a provide section so that the initial dependency() "just works" by falling back to the subproject.